EIPS/eip-7800.md

```diff
@@ -0,0 +1,220 @@
+---
+eip: 7800
```

Comment:
```suggestion
eip: 7804
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.